modules/index.js

```diff
@@ -15,3 +15,6 @@ export StaticRouter from './StaticRouter'
 
 // Util for server rendering "pre-render match"
 export matchPattern from './matchPattern'
+
+// Util for server rendering context
```

Comment:
I missed how to do this in the docs (don't know how!) and I only knew this was possible because of that comment 😂